### PR TITLE
Test the Trust project option

### PR DESCRIPTION
**Pull request by rldhont**
In QGIS3, the user can add an option to force to trust the project about extent and primarykey in PostgreSQL/Postgis layers
https://github.com/qgis/QGIS/pull/5094

### CODE REVIEW

**Comment by rldhont** [comment]
Hi @yjacolin what do you think about this PR ?

File: docker-compose.yml [hunk 1]
```diff
@@ -54,6 +54,21 @@ qgis3:
   links:
     - db
 
+qgis3-trust:
+  image: camptocamp/qgis-server:3
+  environment:
+    QGIS_SERVER_LOG_LEVEL: 3
+    QGIS_OPTIONS_PATH: /root/
+    QGIS_SERVER_PARALLEL_RENDERING: "true"
+    QGIS_SERVER_MAX_THREADS: 8
```
Comment (pvalsecc) [inline docker-compose.yml:63]: I doubt those 3 last variables reach QGIS. Apache is not forwarding automatically the environment variables to the FCGI daemon. You need to add stuff like that in the QGIS configuration:
https://github.com/camptocamp/docker-qgis-server/blob/master/server/runtime/etc/apache2/conf-enabled/qgis.conf#L13

**Review by pvalsecc — CHANGES_REQUESTED** [review]
And you need to add stuff there:
* https://github.com/camptocamp/ms_perfs/blob/master/perfs/run.sh#L27
* https://github.com/camptocamp/ms_perfs/blob/master/perfs/run.sh#L33

File: docker-compose.yml (hunk 1 above)
Comment (rldhont) [reply docker-compose.yml:63]: These lines are exactly the same as in qgis3 sections. I have just copy/paste the configuration.

The link you give is not reachable, so I don't know how to do so.

Comment (yjacolin) [reply docker-compose.yml:63]: This link is private :( but what about adding these variables in the Apache conf? I can do it if you agree.

**Comment by rldhont** [comment]
@pvalsecc thanks for the review, I'll update the run.sh.

File: docker-compose.yml (hunk 1 above)
Comment (pvalsecc) [reply docker-compose.yml:63]: Sorry didn't remember that our QGIS image build was top secret :-(
I'm OK with adding those variables.

Comment (yjacolin) [reply docker-compose.yml:63]: @rldhont I will do this week and let you know as soon as it does.

Comment (pvalsecc) [reply docker-compose.yml:63]: @yjacolin, let me do it. I have other things to sneak in there.

**Comment by rldhont** [comment]
@pvalsecc the run file has been updated

**Review by pvalsecc — APPROVED** [review]

**Comment by pvalsecc** [comment]
@yjacolin, if you are OK with that, please merge.

**Comment by pvalsecc** [comment]
The fix for QGIS is there: https://github.com/camptocamp/docker-qgis-server/pull/6
Will need to be ported to the 3.0 and 2.6 branches.

**Comment by yjacolin** [comment]
@pvalsecc 2.16 you meant :) and 2.18 also (next/current LTR).

**Comment by rldhont** [comment]
Thanks!

**Pull request merged**